cmd/service-controller/controller.go

```diff
@@ -309,15 +320,24 @@ func (c *Controller) checkServiceFor(desired *ServiceBindings, actual *corev1.Se
 	if len(actual.Spec.Ports) > 0 {
 		if actual.Spec.Ports[0].Port != int32(desired.publicPort) {
 			update = true
+			if !hasOriginalPort(*actual) {
+				actual.ObjectMeta.Annotations[types.OriginalPortQualifier] = strconv.FormatInt(int64(actual.Spec.Ports[0].Port), 10)
```

Comment:
Do we ever modify the port of the service? (I don't think so).

Reply:
I do not think so. Should I remove this (as well as the "OriginalPortQualifier")?

Reply:
I think it would be clearer not to record an 'original port' if we never change it.

Reply:
Removed